website_blog_location/__openerp__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Locations in website blogs",
+    "version": "8.0.1.0.0",
+    "author": "Therp BV,Odoo Community Association (OCA)",
```

Comment:
Add also `Website` as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)